bedrock/mozorg/templates/mozorg/internet-health.html

```diff
@@ -0,0 +1,243 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
+
+{% block page_title %}{{ _('Keeping the Internet healthy') }}{% endblock %}
+
+{% block page_desc %}{{ _('The Internet is our largest shared global resource.') }} {{ _('Help us keep it healthy, open and safe.') }}{% endblock %}
+
+{% block body_id %}internet-health{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'internet-health' %}
+{% endblock %}
+
+{% block site_header %}
+  {% include 'mozorg/home/includes/nav.html' %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="main-heading-container">
+    <div class="content">
+      <header class="main-heading">
+        <h1><span>{{ _('A healthy Internet needs all of us') }}</span></h1>
```

Comment:
There may well be a perfectly valid reason for needing the `<span>` in the headings, but could the background color be set directly on the `<h1>` if we then changed it to `display: inline-block;`?